### PR TITLE
[ci skip]Chinese annotation

### PR DESCRIPTION
**Pull request by VisualSJ**
CCEventMouse.h
CCEventTouch.h

### CODE REVIEW

File: cocos/base/CCEventMouse.h [hunk 1]
```diff
@@ -45,14 +45,19 @@
 
 NS_CC_BEGIN
 
-/** @class EventMouse
- * @brief The mouse event.
+/** @class @~english EventMouse
+ * @~chinese 鼠标事件对象
```
Comment (pandamicro) [inline cocos/base/CCEventMouse.h]: @class 不需要翻译

File: cocos/base/CCEventMouse.h [hunk 2]
```diff
@@ -64,34 +69,50 @@ class CC_DLL EventMouse : public Event
...
+     * @param scrollY @~english The scroll data of y axis.
+     * @~chinese y轴滚动的数据
      */
     inline void setScrollData(float scrollX, float scrollY) { _scrollX = scrollX; _scrollY = scrollY; };
-    /** Get mouse scroll data of x axis.
+
+    /** @~english Get mouse scroll data of x axis.
      * 
-     * @return The scroll data of x axis.
+     * @~chinese
```
Comment (pandamicro) [inline cocos/base/CCEventMouse.h]: Please add chinese

File: cocos/base/CCEventMouse.h [hunk 3]
```diff
@@ -107,67 +128,105 @@ class CC_DLL EventMouse : public Event
...
+     * @param button @~english a given mouse button.
+     * @~chinese 鼠标按钮
+     *
      * @js setButton
      */
     inline void setMouseButton(int button) { _mouseButton = button; };
-    /** Get mouse button.
+
+    /** @~english Get mouse button.
+     * @~chinese 获取一个鼠标按钮
```
Comment (pandamicro) [inline cocos/base/CCEventMouse.h]: "获取触发事件的鼠标按键"

File: cocos/base/CCEventMouse.h [hunk 4]
```diff
@@ -107,67 +128,105 @@ class CC_DLL EventMouse : public Event
...
-     * @return The x coordinate of cursor position.
+     * @return @~english The x coordinate of cursor position.
+     * @~chinese 光标的x轴坐标
      * @js getLocationX
      */
     inline float getCursorX() const { return _x; };
-    /** Get the cursor position of y axis.
+
+    /** @~english Get the cursor position of y axis.
+     * @~chinese 获取光标的x轴坐标
```
Comment (pandamicro) [inline cocos/base/CCEventMouse.h]: Y axis

File: cocos/base/CCEventMouse.h [hunk 5]
```diff
@@ -107,67 +128,105 @@ class CC_DLL EventMouse : public Event
...
+     *
+     * @return @~english The y coordinate of cursor position.
+     * @~chinese x轴坐标
```
Comment (pandamicro) [inline cocos/base/CCEventMouse.h]: Y axis

File: cocos/base/CCEventTouch.h [hunk 6]
```diff
@@ -39,15 +39,18 @@ class Touch;
 
 #define TOUCH_PERF_DEBUG 1
 
-/** @class EventTouch
- * @brief Touch event.
+/** @class @~english EventTouch
+ * @~chinese 触摸事件对象
```
Comment (pandamicro) [inline cocos/base/CCEventTouch.h]: @class don't need to be translated

File: cocos/base/CCEventTouch.h [hunk 7]
```diff
@@ -56,33 +59,43 @@ class CC_DLL EventTouch : public Event
...
+     * @~chinese 设置事件类型代码
+
+     * @param eventCode @~english A given EventCode.
+     * @~chinese 触摸事件所对应的事件类型
      */
     void setEventCode(EventCode eventCode) { _eventCode = eventCode; };
-    /** Set the touches
+
+    /** @~english Set the touches
+     * @~chinese 设置触摸列表
```
Comment (pandamicro) [inline cocos/base/CCEventTouch.h]: 触摸点

File: cocos/base/CCEventTouch.h [hunk 8]
```diff
@@ -56,33 +59,43 @@ class CC_DLL EventTouch : public Event
...
      *
-     * @param touches A given touches vector.
+     * @param touches @~english A given touches vector.
+     * @~chinese 一个触摸列表
```
Comment (pandamicro) [inline cocos/base/CCEventTouch.h]: 触摸点

**Comment by natural-law** [comment]
@andyque 
This PR is ok for merge.

**Pull request merged**